packages/editor/src/components/page-attributes/parent.js

```diff
@@ -122,8 +122,7 @@ export function PageAttributesParent() {
 		const opts = getOptionsFromTree( tree );
 
 		// Ensure the current parent is in the options list.
-		const optsHasParent = find(
-			opts,
+		const optsHasParent = opts.find(
```

Comment:
`opts` will always be an array:

https://github.com/WordPress/gutenberg/blob/2dea6e8aee5a8ed5f0ee46769a28e7380188a244/packages/editor/src/components/page-attributes/parent.js#L108